documentation/book/proc-user-operator-with-resource-requests-limits.adoc

```diff
@@ -0,0 +1,48 @@
+// Module included in the following assemblies:
+//
+// assembly-deploying-the-user-operator.adoc
+
+[id='proc-user-operator-with-resource-requests-limits-{context}']
+= Configuring the User Operator with resource requests and limits
+
+.Prerequisites
+
+* A running Cluster Operator
+
+.Procedure
+
```

Comment:
_Please begin the procedure with the following step._
1. Update the Kafka cluster configuration in an editor, as required:
 On Kubernetes, use:
 `kubectl edit kafka my-cluster`
 On OpenShift, use:
`oc edit kafka my-cluster`

_Put `my-cluster` in italics._

Reply:
^ This is reused from @PaulRMellor 's recent enhancements to the upgrade chapter. I think we should adopt it throughout the documentation.